### DISCUSSION TITLE
Possibility to increasse the number of lines the log panel shows on elasticsearch datasource

### DISCUSSION THREAD

**Discussion by pbuentam** [grafana/grafana#58404] [Data Sources]
**What would you like to be added**:
A parameter so the number of lines shown by the logs panel with an elasticsearch datasource or the possibility to scroll through the results, maybe using the search after API. 

**Why is this needed**:
In this moment it is limited to 500 and we have logs lots of lines in any give time. For example Power Center has a precission of seconds and at any given second we have log files with more than 10k lines, QlikView has a simmilar problem, both of them with logging set to INFO.

**Comment by ivanahuckova** [comment]
@Elfo404 I have removed here `are/panel/logs` label and added `datasource/Elasticsearch` as it is related directly to data source settings. 

Also, as a (hacky) workaround, could we just use Raw Data query with Logs Panel? As for Raw Data query, we can customize size.

**Comment by Elfo404** [comment]
@ivanahuckova a PR to add a `size` option to the logs would be quite simple, but unfortunately, it wouldn't be the proper solution as by default the `size` option is limited to `10000`.

the proper way to support this, as pointed out by @pbuentam would be to use the search after API.

What I'd suggest is to add a `limit` option to the logs metric type. In a first moment we can  map it to the `size` param as we currently do with `raw_data` and later on, when we figure out a way to implement "pagination" in a generic way, we can use it to perform multiple search requests until we reach the limit. WDYT?

**Comment by gabor** [comment]
there currently is a limit option in the logs-mode, so the limit can be increased from the default 500 to whatever elastic allows (10000 by default).

to support more than 10000 is the topic of https://github.com/grafana/grafana/discussions/56255 too, so i will close this discussion as a duplicate.
